Rakefile

```diff
@@ -2,7 +2,7 @@ require "foodcritic/version"
 require "bundler"
 require "rake/testtask"
 
-task :default => [:man, :install, :rubocop, :test, :features]
+task :default => [:man, :install, :test, :features, :rubocop]
```

Comment:
This is something lamont brought up today and I was running into it. Style is the thing you care about the least and you often fail tests there first, which stops testing. This way you see your failing specs first and you can clean up your style later. It avoids multiple commits to style.